### ISSUE TITLE
--stateless and thumbprint security (if it's needed) query.

### ISSUE THREAD

**Issue by cds1984** [acmesh-official/acme.sh#3025]
Sorry if this has been asked (oh and great script! love it)

I notice that after using --stateless with the correct configuration on the server to push the thumbprint via the URL it expects, on staging at least and after revoking and removing, it no longer checks the acme-challenge URL and instead reports,
[Fri Jul  3 18:34:27 AWST 2020] <DOMAIN> is already verified, skip http-01.
[Fri Jul  3 18:34:27 AWST 2020] Verify finished, start to sign.

1. Does this mean it only uses the acme-challenge URL as a one hit on the original --issue or ??
2. I can restrict the access to the URL by altering the config before and after the cronjob to check for renewal, but I don't know if it matters or not if the thumbprint is in the wild... what can a malicious actor do with it?

Basically 2 questions.

Thanks.

**Comment by cds1984** [comment]
All good.
No need for a reply, have decided to shy away from the concept.
